src/stock.py

```diff
@@ -0,0 +1,48 @@
+class Stock:
+    name: str
+    shares: int
+    price: float
+    change: float
+
+    def __init__(self, name: str, shares: int, price: float) -> None:
+        self.name = name
+        self.shares = shares
+        self.price = price
+        self.change: float
+
+    def cost(self) -> float:
+        """
+        Calculates the cost of the stock.
+
+        :return: Returns the cost of the stock
+        """
+        return self.shares * self.price
+
+    def sell(self, nshares: int) -> None:
+        """
+        Subtracts shares.
+
+        :param nshares: Number of shares to sell
+        """
+        self.shares -= nshares
+
+    def __eq__(self, other: object) -> bool:
+        """
+        Tests equivalence of two Stock objects.
+
+        :param other: Stock object to compare to
```

Comment:
Not really exact since other can be basically any Python object (for non-Stock object it will just return `NotImplemented`):
```suggestion
        :param other: any Python object to compare to
```

Reply:
Updated.